com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKeyboardPressEvent.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UX
+{
+    [Serializable]
+    public class NonNativeKeyboardPressEvent : UnityEvent<NonNativeKey> { }
```

Comment:
I would probably put all of these events in one file; when doing this for Dialog, I modeled the format + layout after the (excellent) XRI interactable events

https://github.com/needle-mirror/com.unity.xr.interaction.toolkit/blob/master/Runtime/Interaction/XRInteractionEvents.cs (using the Needle mirror for reference)

Reply:
These will also all need documentation

Reply:
Documentation added and moved to single file.